tests/entity_classifier/mock_response.py

```diff
@@ -0,0 +1,68 @@
+mock_input_text1_all_entity_false = """
+    Sachin's SSN is <US_SSN>.
+    ITIN number <US_ITIN>
+    His AWS Access Key is: <AWS_ACCESS_KEY>.
+    And Github Token is: <GITHUB_TOKEN>
+"""
+
+mock_input_text1_all_entity_true = """
```

Comment:
both mock_input_text1_all_entity_false and mock_input_text1_all_entity_true look the same, is this expected?